src/PageContent/AssignVolunteers/AssignVolunteersController.js

```diff
@@ -0,0 +1,236 @@
+import React, { Component } from 'react';
+import './AssignVolunteers.css';
+import Edit from './Edit';
+import Confirmation from './Confirmation';
+import AssignVolunteersIndex from './AssignVolunteersIndex';
+
+class AssignVolunteersController extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            step: 0,
+            deliveries: [],
+            selectedDelivery: -1,
+            s1name: '',
+            s1phone: '',
+            s1email: '',
+            s2name: '',
+            s2phone: '',
+            s2email: '',
```

Comment:
see comment in `Edit.js` about following format in schema to have `deliverer1` and `deliverer2` objects instead.

Reply:
Fixed